packages/application-generic/src/services/cache/key-builders/shared.ts

```diff
@@ -1,45 +1,79 @@
 /**
- * Use this to build a key for entities that are scoped to an environment
+ ***************************************
+ *         BASE KEY BUILDERS
+ ***************************************
+ */
```

Comment:
Maybe we could create a separate file for every section that has such a title, wdyt?